docs/developer-guide/nodejs-api.md

```diff
@@ -550,13 +550,16 @@ In this way, you can retrieve the text and AST used for the last run of `linter.
 
 ### Linter#verifyAndFix()
 
-This method is similar to verify except that it also runs autofixing logic, similar to the `--fix` flag on the command line. The result object will contain the autofixed code, along with any remaining linting messages for the code that were not autofixed.
+This method is similar to `verify` except that it also runs autofixing logic, similar to the `--fix` flag on the command line. The result object will contain the autofixed code, along with any remaining linting messages for the code that were not autofixed.
+
+The default parser `espree` is set to a default [setting](../user-guide/configuring.md#specifying-parser-options) `ecmaVersion: 5` so if your code is more modern than ES5, remember to set `parserOptions`. When the parser does not recognise the code syntax, it silently fails and does not traverse any AST or emit any nodes; the `linter.verifyAndFix()` result is still returned but it will be a non-fixed, original input.
```

Comment:
I dont think it silently fails ! can you confirm !?

also `setting` -> `options`

Reply:
@anikethsaha thank you for swift review. Speaking strictly in terms of `linter.verifyAndFix`, if parser ECMA version is set wrongly and code is not recognised, `verifyAndFix` will not `throw` ("loud" error) — instead it will return unfixed result — that's what I mean by "failing silently". I spotted this troubleshooting an eslint plugin — ECMA version was not set right (but I didn't know at that time) — ES6 code caused `verifyAndFix` to return unfixed result — it was not throwing (or at least propagating inner parser throwing) to the terminal!

Reply:
I'm happy to remove the paragraph, just I thought, in proper API documentation, this part of `verifyAndFix` behaviour is not properly documented.

Reply:
> I'm happy to remove the paragraph, just I thought, in proper API documentation, this part of `verifyAndFix` behaviour is not properly documented.

I don't think you have to remove the whole para cause it does includes some good points. About the silent thing, I will test and I will get back here.